x/bank/types/metadata.go

```diff
@@ -9,12 +9,21 @@ import (
 )
 
 // Validate performs a basic validation of the coin metadata fields. It checks:
+//  - Name and Symbol are not blank
 //  - Base and Display denominations are valid coin denominations
 //  - Base and Display denominations are present in the DenomUnit slice
 //  - Base denomination has exponent 0
 //  - Denomination units are sorted in ascending order
 //  - Denomination units not duplicated
 func (m Metadata) Validate() error {
+	if strings.TrimSpace(m.Name) == "" {
+		return errors.New("name field cannot be blank")
+	}
+
+	if strings.TrimSpace(m.Symbol) == "" {
```

Comment:
not sure if this always needs to be uppercased and < 4 chars?

Reply:
I'd say fine as is. We should allow teams to do longer than 4. In the case of emojis it will be longer